internal/pkg/service/common/cliconfig/flags.go

```diff
@@ -64,7 +64,8 @@ func flagsFromStruct(config any, fs *pflag.FlagSet, parents []string) error {
 
 		// Get optional usage tag
 		usage := field.Tag.Get("usage")
-		fieldPath := append(parents, partialName)
+		fieldPath := parents
+		fieldPath = append(fieldPath, partialName)
```

Comment:
I think there is no change in functionality.

I think the warning from the linter correctly indicates a problem in the code, and this is not a fix.

Can you please copy here what the linter wrote to you?

-----

In Go, it is not obvious how slices work in detail, try looking at this example.

Guess what the program prints and then run it.

https://go.dev/play/p/iiGLYofQ8N4